### PR TITLE
Added LoginForm and basic Navbar with HomePage.

### PR DESCRIPTION
**Pull request by Bartosz97**
Added basic class structure for Navbar, HomePage and Login. Added Bootstrap v4 and React-Router-Dom.

### CODE REVIEW

**Review by EnjoyTheNoise — COMMENTED** [review]
Remarks:
- please add .vs folder to .gitignore
- what did you do in order to create 3,5k additions and deletions in package-lock?

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 import './App.css';
+import LoginFormContainer from './containers/LoginFormContainer';
+import HomePage from './components/HomePage';
+import Navbar from './components/Navbar';
 
 class App extends Component {
   render() {
     return (
+      <Router>
       <div className="App">
```
Comment (EnjoyTheNoise) [inline src/App.js]: formatting here

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+      <Navbar/>
+        <div className="container content">
+        <Switch>
```
Comment (EnjoyTheNoise) [inline src/App.js]: formatting here

File: src/containers/LoginFormContainer.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+       this.setState({
+           isLoggedIn: true,
+           userID: response.userID,
+           firstName: response.firstName,
+           lastName: response.lastName,
+           email: response.email,
+           birthday: response.birthday,
+           picture: response.picture.data.url       
+       })
+    }
```
Comment (EnjoyTheNoise) [inline src/containers/LoginFormContainer.js]: formatting (no new line)

File: src/containers/LoginFormContainer.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  render() {
+      let fbCotent;
```
Comment (EnjoyTheNoise) [inline src/containers/LoginFormContainer.js]: typo I believe?

Comment (EnjoyTheNoise) [reply src/containers/LoginFormContainer.js]: also gj for using let c:

File: src/containers/LoginFormContainer.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+      if (this.state.isLoggedIn) {
+        fbCotent = (
+            <div style={{
```
Comment (EnjoyTheNoise) [inline src/containers/LoginFormContainer.js]: are we using inline styles or external css files?

**Review by jskurzynska — CHANGES_REQUESTED** [review]
You have to create this PR again, vs files are forbidden here. 
The easiest solution for you - remove this PR, jump to the version of branch without your changes, then create commit with gitignore (.vs should be added there) and only next commit should contain your changes (so probably you have to copy it - it will be the easiest solution)
Then I will check your PR again

**Comment by Bartosz97** [comment]
I need create PR again

**Pull request closed without merging**